docs/install.sh

```diff
@@ -1,15 +1,14 @@
 #!/bin/bash
-mkdir -p /imaging/{bin,conf/rdmp,data} /imaging/db/{mongo,sql} 
-curl -sL https://github.com/HicServices/RDMP/releases/download/v8.0.7/rdmp-8.0.7-cli-linux-x64.tar.xz | tar x --strip-components=1 -JC /imaging/bin/rdmp -f - 
-curl -sL https://github.com/SMI/SmiServices/releases/download/v5.3.0/smi-services-v5.3.0-linux-x64.tgz | tar x --strip-components=1 -zC /imaging/bin/smi -f - 
-mysql --uroot --password=mysqlrootpassword < schema.sql
+mkdir -p /imaging/{bin/rdmp,bin/smi,conf/rdmp,data} /imaging/db/{mongo,sql} 
+curl -sL https://github.com/HicServices/RDMP/releases/download/v8.1.1/rdmp-8.1.1-cli-linux-x64.tar.xz | tar x --strip-components=1 -JC /imaging/bin/rdmp -f - 
+curl -sL https://github.com/SMI/SmiServices/releases/download/v5.4.0/smi-services-v5.4.0-linux-x64.tgz | tar x --strip-components=1 -zC /imaging/bin/smi -f - 
+mysql --user=root --password=mysqlrootpassword < schema.sql
 wget https://raw.githubusercontent.com/HicServices/DicomTypeTranslation/main/Templates/CT.it 
-/imaging/bin/rdmp/rdmp createnew 
-externaldatabaseserver LiveLoggingServer_ID "DatabaseType:MySQL:Server=127.0.0.1;Uid=smi;Pwd=SmiSqlPassword;Database=smi" --dir /imaging/conf/rdmp 
+/imaging/bin/rdmp/rdmp createnew externaldatabaseserver LiveLoggingServer_ID "DatabaseType:MySQL:Server=127.0.0.1;Uid=smi;Pwd=SmiSqlPassword;Database=smi" --dir /imaging/conf/rdmp 
 /imaging/bin/rdmp/rdmp cmd CreateNewImagingDatasetSuite "DatabaseType:MySQL:Name:smi:Server=127.0.0.1;Uid=smi;Pwd=SmiSqlPassword;Database=smi_images" ./data DicomFileCollectionSource CT_ CT.it false true --dir /imaging/conf/rdmp 
 for modality in CR CT DX IO MG MR NM OTHER PT PX RF SR US XA 
 do 
-sed –e "s/MR_/${modality}_/g" <<'EOC' | ~/bin/rdmp/rdmp -f /dev/stdin --dir /imaging/conf/rdmp 
+sed –e "s/MR_/${modality}_/g" <<'EOC' | /imaging/bin/rdmp/rdmp -f /dev/stdin --dir /imaging/conf/rdmp
```

Comment:
On my Ubuntu machine this line is spitting out
"""
INFO Exiting with code -56 .
sed: -e expression #1, char 1: unknown command: `▒'
"""

Reply:
Strange error, hopefully fixed now?